lineage/states/stateCommon.py

```diff
@@ -130,21 +134,27 @@ def gamma_estimator_atonce(gamma_obs, time_cen, gamas, x0=None):
     arrgs = (arg1, arg2, arg3, arg4)
 
     if x0 is None:
-        x0 = np.array([20.0, 2.0, 3.0, 4.0, 5.0])
-
-    A = np.zeros((3, 5))
-    np.fill_diagonal(A[:, 1:], -1.0)
-    np.fill_diagonal(A[:, 2:], 1.0)
+        if phase:
```

Comment:
The constraint isn't about phase-specific versus not, it's about whether the treatments should have a dose-response effect.

Reply:
Understood. I should write a more general code for this. I wrote this just to separate the new data analysis from the phase-specific ones.